README.md

```diff
@@ -12,21 +12,21 @@ response status, rendering validation errors, and inserting changesets.
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed by:
 
-  1. Add ja_resource to your list of dependencies in `mix.exs`:
+  1. Adding ja_resource to your list of dependencies in `mix.exs`:
```

Comment:
So, "the package can be installed as: 1) Add .. 2) Ensure" doesn't work grammatically. This change does work grammatically, but if you want it to be explicitly directive in tense (keeping it as Add / Ensure) then I'd suggest something like:

"the package can be installed with the following steps: 1) Add.. 2) Ensure.."